### DISCUSSION TITLE
Need hints to develop with textual without constant stop/restart of app to pickup code changes

### DISCUSSION THREAD

**Discussion by reckoner** [Textualize/textual#4201] [Q&A]
Hi, I'm trying to develop my first textual app and I'm finding it tedious to keep stopping and starting my app when I make changes. When I develop with fastapi, it is possible to set an "auto-reload" setting to pick up my changes as I code. Is there something like this in textual

Thanks for the great work!

**Comment by davep** [comment]
You can get [live reload of styles if using an external stylesheet and the devtools](https://textual.textualize.io/guide/devtools/), but there isn't anything built into Textual that will reload your whole application on any change (I have seen people document at least one approach this on the Textual discord server, but I'm afraid I can't remember how they went about it; but it seems there are approaches you could take).
